CesiumRasterOverlays/src/GeoJsonDocumentRasterOverlay.cpp

```diff
@@ -159,36 +170,46 @@ struct GlobeRectangleFromObjectVisitor {
           Cartographic::fromDegrees(point.x, point.y));
     }
   }
-  void operator()(const GeoJsonLineString& line) {
-    for (const glm::dvec3& point : line.coordinates) {
-      builder.expandToIncludePosition(
-          Cartographic::fromDegrees(point.x, point.y));
+  void visitWithLineWidth(
+      const std::vector<glm::dvec3>& coordinates,
+      const std::optional<LineStyle>& lineStyle) {
+    if (lineStyle && lineStyle->widthMode == LineWidthMode::Meters) {
+      const double halfWidth =
+          (lineStyle->width / ellipsoid.getRadii().x) / 2.0;
+      for (const glm::dvec3& point : coordinates) {
+        // Include the width of the line in the bounding region calculation
+        builder.expandToIncludePosition(Cartographic::fromDegrees(
```

Comment:
Maybe you already did, but it's worth testing that this logic still works correctly when adding +/- halfWidth causes an antimeridian wrap. I'm actually not sure if `expandToIncludePosition` will handle that correctly or not.